src/potential/gmtgravmag3d.c

```diff
@@ -241,7 +241,13 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
 	GMT_Usage (API, -2, "Set z level of reference plane [Default = 0].");
 	GMT_Option (API, "bi");
 	GMT_Usage (API, 1, "\n-fg Converts geographic grids to meters using a \"Flat Earth\" approximation.");
-	GMT_Option (API, "h,i,o,r,:,.");
+	GMT_Option (API, "h,i,o,r");
+#ifdef HAVE_GLIB_GTHREAD
+	GMT_Option (API, "x");
+#else
+	GMT_Usage (API, 1, "\n-x Not available since this binary was not build with multi-threading support.");
```

Comment:
Maybe make this message more clear that we are talking of GLIB muti-threading and not OMP.

Reply:
OK, done